float64 ang_vel

